index.html

```diff
@@ -1,13 +1,89 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Example</title>
+        <title>LAB 2</title>
+        <meta name="Natalya Jones" content="lab2">
+        <meta charset="UFT-8">
+        <style>
+            table, th, td {
+                border: 2px solid black;
+                border-collapse: collapse;
+            }
+        </style>
+        <style>
+            td {text-align: center;}
+        </style>
     </head>
     <body>
-        <h1>Title header: NMJ's edit :) </h1>
-        <!-- <h4>This is a smaller header!</h4> -->
-        <h4>Foods I borught with me to school today</h4>
-        <!-- <p>This is an example of a simple page with one paragraph.</p> -->
-        <p>I brough banana bread, a breakfast sandwich, and greek chicken and rice</p>
+        <h1> Hi, my name is Natalya Jones</h1>
+        <h2> Facts about me</h2>
+        <ul>
+            <li> I am in my <strong>second</strong> year of computer science
+            <li> I was born in <em>Fredericton New Brunswick</em>
+            <li> My family moved to Sherbrooke in 2012 because my dad got a job at <b>Bishops</b>
+                <ul>
+                    <li> He teaches math
+                </ul>
+            <li> After university, I do not know what job I will do
+        </ul>
+        <table>
+            <tc>
+                <th>School Attended </th>
+                <th>Grade (if applicable) </th>
+                <th>Province </th>
+            </tc>
+            <tr>  
+                <td> Connaught Street Elementary</td>
+                <td> K-1 </td>
+                <td> NB</td>
+            </tr>
+            <tr>  
+                <td> Bayview Elementary</td>
+                <td> 2</td>
+                <td> NB</td> 
+            </tr>
+            <tr>  
+                <td> Forest Hills </td>
+                <td> 3</td>
+                <td> NB</td>
+            </tr>
+            <tr>  
+                <td> Lennoxville Elementary </td>
+                <td> 4-6</td>
+                <td> QC</td>
+            </tr>
+            <tr>  
+                <td> Alexander Galt RHS </td>
+                <td> 7-11</td>
+                <td> QC</td>
+            </tr>
+            <tr>  
+                <td> Champlain RC </td>
+                <td> 2020-24</td>
+                <td> QC</td>
+            </tr>
+            <tr>  
+                <td> Bishops University </td>
+                <td> 2024-current</td>
+                <td> QC</td>
+            </tr>
+        </table>
+        <br>
```

Comment:
This is not exactly how we usually use the `<br>` tag. To add more space between your table and the link, we would rather play with the attributes of the tables, adding a margin or padding around it.

The `<br>` is more about splitting some text in the middle. Ex:
```
This is a <br>
good usage of <br>
line breaks
```